### PR TITLE
Add EpiNow2 submission

### PR DESCRIPTION
**Pull request by seabbs**
Adds an automated submission for EpiNow2. Rather than using a submodule this action simply downloads the forecast from a dated URL and then copies the other parts of the workflow to PR this into the hub repository. 

Note I wrote this using github.dev so likely a bit dodgy. Happy to improve under direction (for example writing this in a script vs inline in the action).

### CODE REVIEW

File: .github/workflows/automate_epinow2.yml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    - name: Install system dependencies
+      run: sudo apt-get install libudunits2-dev libcurl4-openssl-dev libgdal-dev libv8-dev
+
+    - name: Install R dependencies
+      run: |
+       install.packages(c("fread", "glue"))
+      shell: Rscript {0}
+
+    - name: Generate forecasts
+      run: |
```
Comment (Bisaloo) [inline .github/workflows/automate_epinow2.yml:30]: What about just

```sh
TODAY=date --iso-8601
wget https://raw.githubusercontent.com/epiforecasts/europe-covid-forecast/master/submissions/rt-forecasts/$TODAY/$TODAY-epiforecasts-EpiNow2.csv
```

?

So we can get rid of the R step and shave off some running time.

**Review by Bisaloo — CHANGES_REQUESTED** [review]
LGTM.

I just requested a simpler (no need to read+write the data when we can just download it) & separate script.

Regarding the suggestion to use bash instead of R, feel free to do as you see fit.

**Comment by seabbs** [comment]
> What about just

```sh
TODAY=date --iso-8601
wget https://raw.githubusercontent.com/epiforecasts/europe-covid-forecast/master/submissions/rt-forecasts/$TODAY/$TODAY-epiforecasts-EpiNow2.csv
```

?

So we can get rid of the R step and shave off some running time. 

Good idea. Have implemented this (not sure have actually set the bash script to run correctly in the action as I think I have never actually done this). Thought about expanding the readme but wasn't sure I was writing what you had in mind so left documenting this out of this PR.

File: README.Rmd [hunk 2]
```diff
@@ -48,4 +48,4 @@ To submit your model for auto-submission, you need to [open an issue](https://gi
 Please note that although encouraged, it is not strictly mandatory to submit the
 code generating your forecasts. You can submit a script fetching already 
 existing forecasts from another location such as an API, another GitHub 
-repository, a data repository, etc.
+repository, a data repository, etc.∂
```
Comment (Bisaloo) [inline README.Rmd:51]:
```suggestion
repository, a data repository, etc.
```

File: .github/workflows/automate_epinow2.yml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+    
+    - uses: r-lib/actions/setup-r@v1
+      with:
+        use-public-rspm: true
+
+    - name: Install system dependencies
+      run: sudo apt-get install libudunits2-dev libcurl4-openssl-dev
```
Comment (Bisaloo) [inline .github/workflows/automate_epinow2.yml:18]: Do we need this at all?

File: .github/workflows/automate_epinow2.yml [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+    - name: Generate forecasts
+      run: data-submission/epinow2/main.sh
+      shell: bash
+      
+    - uses: actions/upload-artifact@v2
+      with:
+        name: epinow2
+        retention-days: 5
+        path: data-submission/epinow2
```
Comment (Bisaloo) [inline .github/workflows/automate_epinow2.yml:28]:
```suggestion
        path: data-submissions/epinow2
```

File: .github/workflows/automate_epinow2.yml (hunk 3 above)
Comment (seabbs) [reply .github/workflows/automate_epinow2.yml:18]: Good point.

**Comment by seabbs** [comment]
Thanks @Bisaloo,updated based on your comments + turned off downloading submodules as not to used here.

**Pull request merged**